zipline-loader/src/jvmTest/kotlin/app/cash/zipline/loader/OkHttpZiplineHttpClientTest.kt

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Could move to jniTest to run on both JVM and Android

Reply:
Fixed.